api/routes/tasks.py

```diff
@@ -79,4 +131,15 @@ def delete_task(user_id):
     # body has task id
     body = request.get_json()
 
-    db.delete(Task).where(Task.user_id == user_id and Task.id == body["id"])
+    task = (
+        db.session.query(Task)
+        .where(Task.user_id == user_id, Task.task_id == body["id"])
+        .one()
```

Comment:
Is this `one` predicate needed? `task_id` is the primary key of the `Task` table, so we should only ever get back one result?

Reply:
this simply asserts that there is only one task